### ISSUE TITLE
The Chisel Bit Storage is very broken when used with block bits

### ISSUE THREAD

**Issue by aaronhowser1** [ChiselsAndBits/Chisels-and-Bits#686]
For any bug reports please provide the following details as well,
* MC Version: 1.16.5
* C&B Version: chiselsandbits-0.3.4-RELEASE
* Do You have Optifine: No

![image](https://user-images.githubusercontent.com/15698722/121638306-5cea8580-ca50-11eb-8d45-3233bf6acaba.png)
Right-click one of the stacks into the storage
![image](https://user-images.githubusercontent.com/15698722/121638328-6673ed80-ca50-11eb-872f-2c40ef1ca1bd.png)
It now has the nbt: `x: 197, bits: 64, y: 22, z: -24, fluid: "", id: "chiselsandbits:bit_storage", blockstate: 1`
Doesn't mention that it's stone at all.

Right-clicking more just takes the stone bits back out. I can't reproduce it, but for a while it wasn't even taking the stone, just causing desync issues (on singleplayer). The bits would vanish from my inventory, but hitting Q would drop them anyway.

![image](https://user-images.githubusercontent.com/15698722/121638493-b2269700-ca50-11eb-889b-9df3cc385443.png)
Breaking it with an axe voids all the Bits inside.

Basically, all the non-fluid parts of it are jank as heck. Bit storage should probably be moved to its own thing, separate from the Bit Tank. I know this says it's Bit Storage, but the only functioning part is liquid bits.

**Comment by marchermans** [comment]
Should be fixed in the newest beta.
